src/reporting/ReportingWindow.sol

```diff
@@ -289,4 +291,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable {
         numberOfReportsByMarket[_market] += 1;
         return true;
     }
+
+    function getWinningPayoutDistributionHashFromFork(Market _market) constant returns (bytes32) {
```

Comment:
From market you could just do `MarketExtensions(controller.lookup('MarketExtensions')).getWinningPayoutDistributionHash(this)`